Week1/homework/exec_query.js

```diff
@@ -0,0 +1,34 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: '7566',
+  database: 'world'
+});
+
+connection.connect();
+
+const select_query = [
+  `SELECT name FROM country WHERE population >= 8000000;`,
+  `SELECT name FROM country WHERE name LIKE '%land%'`,
+  `SELECT name, population FROM city WHERE population BETWEEN 500000 AND 1000000`,
+  `SELECT name FROM country WHERE continent LIKE '%Europe%'`,
+  `SELECT name FROM country ORDER BY surface_area_km2 DESC`
+];
+
+for (let i of select_query) {
```

Comment:
Extra challenge: can you make this function to accept an input from the user, say, a number, and then give the requested answer? For example, if I type 2, I'm expecting the second question from the homework.